pkg/helmexec/exec.go

```diff
@@ -606,10 +607,12 @@ func (helm *execer) info(out []byte) {
 	}
 }
 
+// Write to writer if not nil, otherwise to logger.
 func (helm *execer) write(w io.Writer, out []byte) {
```

Comment:
`write` is literally for writing to a Writer so that the consumer of helmfile can consume it.
So, I'd say calling `helm.logger.Infof` from `write` in inappropriate and complicates the behavior without clear benefit? 🤔 

Can you perhaps just update `helm.write(nil, out)` with `helm.info(out)` in places where you feel appropriate, and leave this `write` function as-is?

Reply:
Makes sense, I only discovered this info function halfway through the PR.
You are right, the write function change is actually not required for this PR to work.